lib/index.js

```diff
@@ -138,7 +144,8 @@ module.exports = {
           flattendPackageList[dep.name] = flattendPackageList[dep.name] || dep;
         });
       } catch (error) {
-        // could not read installed packages folder
+        // could not read installed packages folder,
+        // continue only with manifest list
```

Comment:
Maybe let the user know we're running on the manifest file only? Not sure if plugins should be using `console.log`, seems more like the cli responsibility. Maybe mention this fact on the object returned from `inspect`?